src/core/tracking.js

```diff
@@ -168,10 +170,18 @@ export class Tracking {
 }
 
 export function getAddonTypeForTracking(type) {
+  const extensionTrackingTypes = [
+    ADDON_TYPE_DICT,
+    ADDON_TYPE_EXTENSION,
+    ADDON_TYPE_LANG,
+    ADDON_TYPE_OPENSEARCH,
+  ];
+
+  const index = extensionTrackingTypes.indexOf(type);
+
   return (
     {
-      [ADDON_TYPE_EXTENSION]: TRACKING_TYPE_EXTENSION,
-      [ADDON_TYPE_OPENSEARCH]: TRACKING_TYPE_EXTENSION,
+      [extensionTrackingTypes[index]]: TRACKING_TYPE_EXTENSION,
```

Comment:
Could we simply add all the constants in this map? I find confusing the use of `index`. I think it is fine to list all the constants:

```
{
  ...
  [ADDON_TYPE_DICT]: TRACKING_TYPE_EXTENSION,
  [ADDON_TYPE_EXTENSION]: TRACKING_TYPE_EXTENSION,
  ...
}
```